src/components/photoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import "./PhotoGallery.scss"
+
+export const PhotoGallery: React.FunctionComponent = () => {
+
+  const [imageUrls, setImageUrls] = useState<string[]>();
+  const img_urls: string[] = [];
+
+        useEffect(() => {
+          const url = `https://api.nasa.gov/planetary/apod?count=50&api_key=DEMO_KEY`;
+          
+          const fetchData = async () => {
+            try {
+              const response = await fetch(url);
+              const json = await response.json();
+              json.forEach((item: {url: string}) => img_urls.push(item.url));  
+              setImageUrls(img_urls);
+            } catch (error) {
+              console.log("error", error);
+            }
+          };
+      
+          fetchData();
+      }, []);
+
+  return (
+      <div className = "imageGallery">
+      {imageUrls?.map(url => <img className="spaceImage" src={url} alt = ""/>)}
```

Comment:
It's good UX to display something here while the items are loading, for example:
```suggestion
      {
        imageUrls === undefined
          ? <p>Loading...</p>
          : imageUrls.map(url => <img className="spaceImage" src={url} alt = ""/>)
      }
```
(this uses the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator), but you can use an if/else like we did for other code blocks if that's easier to understand!)